packages/prop-controllers/src/table/table.ts

```diff
@@ -0,0 +1,130 @@
+import { z } from 'zod'
+import {
+  ControlDataTypeKey,
+  CopyContext,
+  Options,
+  Types,
+} from '../prop-controllers'
+import { P, match } from 'ts-pattern'
+
+const tableDataSchema = z.string()
+
+export type TableData = z.infer<typeof tableDataSchema>
```

Comment:
Why don't we name this `TableId`?

Reply:
Yeah, I thought about that too. Changed.